R/bm-df-to-table.R

```diff
@@ -6,8 +6,13 @@
 #' * `source` A known-file id to use (it will be read in to a data.frame first)
 #'
 #' @export
-df_to_table <- Benchmark("df_to_table",
-  setup = function(source = names(known_sources)) {
+df_to_table <- Benchmark("dataframe-to-table",
+  setup = function(source = c("chi_traffic_2020_Q1",
+                              "type_strings",
+                              "type_dict",
+                              "type_integers",
+                              "type_floats",
+                              "type_nested")) {
```

Comment:
Very minor style nit, though this could in principle go both ways I find the following _slightly_ easier to read:

```suggestion
  setup = function(source = c(
    "chi_traffic_2020_Q1",
    "type_strings",
    "type_dict",
    "type_integers",
    "type_floats",
    "type_nested"
    )) {
```

Reply:
Yeah, that's black style, kind of. It might look weird if there were more than one parameter, though; are we ok with 

```r
  setup = function(
    source = c(
      "chi_traffic_2020_Q1",
      "type_strings",
      "type_dict",
      "type_integers",
      "type_floats",
      "type_nested"
    )
  ) {
    source <- ensure_source(source)
```
?

Reply:
Yup, that's fine too. Mostly I try to avoid the more-indented-than-the-block-should-be-otherwise when I can (though I know it makes sense for function definitions and why RStudio does what it does automatically here...)